### DISCUSSION TITLE
Disabling showed area

### DISCUSSION THREAD

**Discussion by KyL0vV** [streetcomplete/StreetComplete#5370] [Ideas]
It would be nice to be able to disable a formerly downloaded area or better a piece of to not to be overwhelmed with queries you do not want to work on (saves mental energy and advances focus on some chosen tasks!).

Thanks.

**Comment by mnalis** [comment]
- if you go into `Settings` and choose `Delete cache`, it  will _remove formerly downloaded areas_. Is this what you are looking for?
- for  _focusing on some chosen tasks_, I'd recommend presets. You go into `Settings` / `Manage presets` and use `+` button to create new preset, and then in `Settings` / `Quest selection and display order` you can select only some important quests you want to focus on (e.g. I have created new `bicycle` preset which has only bicycle-related quests enabled). You can create and customize several different presets, and then later easily switch between them in `Manage presets` to choose which one you want to use at that moment.
